src/components/layout/LineUpHeadlinerRight.tsx

```diff
@@ -0,0 +1,16 @@
+const LineUpHeadlinerRight: React.FC = () => {
+  return (
+    <div className='flex md:w-2/5 bg-[#1D6858] h-[500px] green-shadow overflow-hidden'>
+      <div className='relative'>
+        <div className='absolute bottom-6 left-0 text-white px-2 font-bold '>
+          <span className='text-xl xl:text-2xl'>???</span>
+        </div>
+        <div className='absolute bottom-0 left-0 text-white px-2 text-lg'>
+          ???
```

Comment:
is this intended?

Reply:
Yes, but it was just tmp file (until I had the last graphic assets for both Headliners)